src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+
+#: src/view/com/profile/ProfileMenu.tsx:336
+msgid "Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you."
+msgstr "Geblokkeerde accounts kunnen niet reageren op uw gesprekken, u niet vermelden en niet op een andere manier met u communiceren."
+
+#: src/view/screens/ModerationBlockedAccounts.tsx:116
+msgid "Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you. You will not see their content and they will be prevented from seeing yours."
+msgstr "Geblokkeerde accounts kunnen niet reageren op uw gesprekken, u niet vermelden en niet anderszins met u communiceren. U ziet hun content niet en zij kunnen de uwe niet zien."
+
+#: src/view/com/post-thread/PostThread.tsx:412
+msgid "Blocked post."
+msgstr "Geblokkeerd bericht."
+
+#: src/screens/Profile/Sections/Labels.tsx:173
+msgid "Blocking does not prevent this labeler from placing labels on your account."
+msgstr "Blokkeren voorkomt niet dat deze labeler labels op uw account plaatst."
+
+#: src/view/screens/ProfileList.tsx:744
+msgid "Blocking is public. Blocked accounts cannot reply in your threads, mention you, or otherwise interact with you."
+msgstr "Blokkeren is openbaar. Geblokkeerde accounts kunnen niet reageren op uw gesprekken, u vermelden en niet anderszins met u communiceren."
+
+#: src/view/com/profile/ProfileMenu.tsx:333
+msgid "Blocking will not prevent labels from being applied on your account, but it will stop this account from replying in your threads or interacting with you."
+msgstr "Blokkeren voorkomt niet dat er labels op uw account worden toegepast, maar het zorgt er wel voor dat dit account niet meer op uw gesprekken kan reageren en niet meer met u kan communiceren."
+
+#: src/view/com/auth/SplashScreen.web.tsx:172
+msgid "Blog"
+msgstr "Blog"
+
+#: src/view/com/auth/server-input/index.tsx:86
+#: src/view/com/auth/server-input/index.tsx:88
+msgid "Bluesky"
+msgstr "Bluesky"
+
+#: src/view/com/auth/server-input/index.tsx:151
+msgid "Bluesky is an open network where you can choose your hosting provider. If you're a developer, you can host your own server."
+msgstr "Bluesky is een open netwerk waar u uw hostingprovider kunt kiezen. Als u een ontwikkelaar bent kunt u uw eigen server hosten."
+
+#: src/components/ProgressGuide/List.tsx:55
+msgid "Bluesky is better with friends!"
+msgstr "Bluesky is leuker met vrienden!"
+
+#: src/components/StarterPack/ProfileStarterPacks.tsx:283
+msgid "Bluesky will choose a set of recommended accounts from people in your network."
+msgstr "Bluesky kiest een aantal aanbevolen accounts van mensen in uw netwerk."
+
+#: src/screens/Moderation/index.tsx:571
+msgid "Bluesky will not show your profile and posts to logged-out users. Other apps may not honor this request. This does not make your account private."
+msgstr "Bluesky toont uw profiel en berichten niet aan afgemelde gebruikers. Andere apps honoreren dit verzoek mogelijk niet. Dit maakt uw account niet privé."
```

Comment:
I think the translation is correct, but the intent is "not signed in", not "signed out" which is the state after you were signed in but no longer, distinct from someone who was never logged in.

```suggestion
msgstr "Bluesky toont uw profiel en berichten niet aan bezoekers die niet ingelogd zijn. Andere apps honoreren dit verzoek mogelijk niet. Dit maakt uw account niet privé."
```

Reply:
Tricky one. Intentionally used "aangemeld". Better would be "niet aan bezoekers die niet aangemeld zijn", but double negative. Therefore chosen for "afgemelde gebruikers", since the original sentence also uses "users" instead of for instance "visitor" to also include visitors that never logged in.

Left unchanged.

Reply:
Yeah I personally find this really hard to translate as well.